examples/platform/linux/BUILD.gn

```diff
@@ -67,6 +67,10 @@ source_set("commodity-price-test-event-trigger") {
   sources = [ "${chip_root}/src/app/clusters/commodity-price-server/CommodityPriceTestEventTriggerHandler.h" ]
 }
 
+source_set("electrical-grid-conditions-test-event-trigger") {
+  sources = [ "${chip_root}/src/app/clusters/electrical-grid-conditions-server/ElectricalGridConditionsTestEventTriggerHandler.h" ]
```

Comment:
why do we do this all over here?

These should be a source set if they are stand-alone.

The error says these are already part of some targets.